src/nlp/commands/dummy_data.py

```diff
@@ -90,8 +90,8 @@ def run(self):
             split_names = ", ".join(split_names)
             if len(files_to_create) > 0:
                 # no glob.glob(...) in `_generate_examples(...)`
-                if len(files_to_create) == 1 and files_to_create[0] == dummy_file_name:
-                    dummy_data_guidance_print += f"- Please create a single dummy data file called '{files_to_create[0]}' from the folder '{dummy_data_folder}'. Make sure that the dummy data file provides at least one example for the split '{split_names}' \n\n"
+                if len(set(files_to_create)) == 1 and files_to_create[0] == dummy_file_name:
```

Comment:
maybe we could use a set directly for `files_to_create` ?

Reply:
yeah good point - changed it to use a set